api/routes.py

```diff
@@ -202,6 +202,13 @@ def annotations():
 def annotation_detail(annotation_id):
     return app.manager.annotations.detail(annotation_id)
 
+@app.route('/annotations/<identifier_type>/<path:identifier>/', methods=['GET'])
```

Comment:
I originally wanted to put this under works, but I didn't want to have to give it a data source and without one it makes the route ambiguous.